### PR TITLE
PTFM-13821 Clean up dx error handling

### PR DESCRIPTION
**Pull request by ivoloshanenko**
https://dnanexus.jira.com/browse/PTFM-13821

### CODE REVIEW

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -1808,21 +1795,18 @@ def get(args):
     elif entity_result_class == 'app':
         get_app(entity_result, args)
     else:
-        parser.exit(3,
-                    'Error: The given object is of class ' +
-                    entity_result['describe']['class'] +
-                    ' but an object of class file, record,' +
-                    ' applet or app was expected\n')
+        err_exit('Error: The given object is of class ' + entity_result['describe']['class'] +
+                 ' but an object of class file, record,' + ' applet or app was expected', 3)
```
Comment (psung) [inline src/python/dxpy/scripts/dx.py]: Consolidate these two string literals into one now that you've unwrapped the lines.

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -1698,9 +1695,9 @@ def get_record(entity_result, args):
         if args.output is None and not args.no_ext:
             filename += '.json'
         if not args.overwrite and os.path.exists(filename):
-            parser.exit(1, fill('Error: path "' +
+            err_exit(fill('Error: path "' +
                                 filename +
```
Comment (psung) [inline src/python/dxpy/scripts/dx.py]: Here and elsewhere, please fix up the indentation.

**Comment by psung** [comment]
A number of the dxclient tests fail since they expected certain commands to fail with the old exit codes. Let me know if you have any questions about reproducing this. Thanks!

**Comment by ivoloshanenko** [comment]
Hi, Phil. Yes, I would be very appreciated for help with tests. I tried it locally, but some of them not working (probably because of local API server). If any way to test this code on Jenkins or some other solution?

**Comment by ivoloshanenko** [comment]
Test file updated. Phil, can you please check tests again? Not of all them I can run home.vdev.

**Comment by psung** [comment]
Looks good, thanks!

**Pull request merged**